### PR TITLE
Update API calls

### PR DESCRIPTION
**Pull request by stuartnelson3**
@juliusv @fabxc can you guys double check the code, and run against a local instance? I checked against pie/gauge/graph, but it wasn't exhaustive.

addresses https://github.com/prometheus/prometheus/issues/1593

TODO:

- Update test fixtures

### CODE REVIEW

File: app/assets/javascripts/angular/services/graph_refresher.js [hunk 1]
```diff
@@ -47,12 +47,12 @@ angular.module("Prometheus.services").factory('GraphRefresher',
...
         return deferred.promise;
       },
-      graphite: function(idx, expression, server, expressionID, endTime, range, step) {
+      graphite: function(idx, expression, server, expressionID, startTime, endTime, step) {
         var deferred = $q.defer();
         $http.get(URLGenerator(server.url, 'render', $scope.vars), {
           params: {
             target: expression,
-            from: GraphiteTimeConverter.graphiteFrom(range, endTime),
+            from: GraphiteTimeConverter.graphiteUntil(startTime),
```
Comment (juliusv) [inline app/assets/javascripts/angular/services/graph_refresher.js]: Why is this `graphiteUntil` now instead of `graphiteFrom`?

Comment (juliusv) [inline app/assets/javascripts/angular/services/graph_refresher.js]: Ah, I guess I get it - but is the `graphiteUntil` function still named well then?

**Comment by juliusv** [comment]
I tested everything I could think of (different widget types, errors, successes, different time ranges), and the only error I've found so far is when you try to change the end time. Then you get:

`endTime.getTime is not a function`

File: app/assets/javascripts/angular/services/graph_refresher.js (hunk 1 above)
Comment (stuartnelson3) [inline app/assets/javascripts/angular/services/graph_refresher.js]: Nope, the name is totally misleading!

**Comment by stuartnelson3** [comment]
@juliusv thanks for checking, like the old days :) anything else you want changed?

**Comment by juliusv** [comment]
Nope! :+1: otherwise. I'm not gonna go into nitpicking mode for basically deprecated code :)

**Comment by stuartnelson3** [comment]
Haha yeah, for definitely deprecated code :)

**Comment by fabxc** [comment]
\o/ great thanks!

**Pull request merged**